src/Compilers/Core/Portable/Operations/Operation.Enumerable.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections;
+using System;
+using System.Diagnostics;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal abstract partial class Operation : IOperation
+    {
+        /// <summary>
+        /// Implements a struct-based enumerable for <see cref="Operation"/> nodes, using a slot-based system that tracks
+        /// the current slot, and the current index in the slot if the current slot is an immutable array. This type is not hardened
+        /// to default instances, and will null reference in these cases. Using <see cref="ChildOperations"/> and this is always
```

Comment:
>Using [](start = 74, length = 5)

Is this a typo? I do not see ```ChildOperations``` used anywhere in this file.

Reply:
No, it's `Operation.ChildOperations`. I'll fully qualify it.

---
In reply to: [541170567](https://github.com/dotnet/roslyn/pull/49804#discussion_r541170567) [](ancestors = 541170567)

Reply:
> No, it's Operation.ChildOperations. I'll fully qualify it.

Still don't see this type **using** it.


---
In reply to: [541171643](https://github.com/dotnet/roslyn/pull/49804#discussion_r541171643) [](ancestors = 541171643,541170567)

Reply:
Moved to Operation.cs.

---
In reply to: [541173933](https://github.com/dotnet/roslyn/pull/49804#discussion_r541173933) [](ancestors = 541173933,541171643,541170567)